lib/public/views/Runs/format/formatRunEnd.js

```diff
@@ -28,17 +28,12 @@ export const formatRunEnd = (run, inline) => {
         return formatTimestamp(run.timeTrgEnd, inline);
     } else if (run.timeO2End) {
         const { date, time } = getLocaleDateAndTime(run.timeO2End);
-        return h(
-            '',
-            h('', date),
-            h(
-                '.flex-row.g2.items-center',
-                [
-                    h('', time),
-                    tooltip(iconWarning(), 'O2 stop is displayed because trigger stop is missing'),
-                ],
-            ),
-        );
+        return h('span', [
+            h('.flex-row.items-center.g2', [
+                `${date} ${time}`,
+                tooltip(iconWarning(), 'O2 stop is displayed because trigger stop is missing'),
+            ]),
+        ]);
```

Comment:
Here you are breaking the display of run overview, where date, time and maybe tooltip are display on 2 lines